payments-core/src/main/java/com/stripe/android/model/SourceParams.kt

```diff
@@ -18,6 +18,8 @@ data class SourceParams internal constructor(
      */
     @SourceType val typeRaw: String,
 
+    internal var typeData: TypeData? = null,
```

Comment:
should we use this in `PaymentIntent` as well?

Reply:
Do you have an example?

Reply:
Sorry I meant the type in `PaymentMethod`, seems [sourceType](https://stripe.com/docs/api/sources/object#source_object-type) and [pmType](https://stripe.com/docs/api/payment_methods/object#payment_method_object-type) are the similar? Seems we do have a [toParamsMap](https://github.com/stripe/stripe-android/blob/34ca5f0e72204fccd1942a3bb55df6638541f636/payments-core/src/main/java/com/stripe/android/model/PaymentMethodCreateParams.kt#L166) for pmType hashes

Reply:
Yeah could consider it for a future PR? This PR is big enough as-is.